### ISSUE TITLE
Remove paragraph on calculating specific intestinal permeability from experimentally measured values

### ISSUE THREAD

**Issue by AndreDlm** [Open-Systems-Pharmacology/docs#120]
In [this section](https://github.com/Open-Systems-Pharmacology/docs/blob/master/part-3/pk-sim-compounds-definition-and-work-flow.md#calculation-of-specific-intestinal-permeabilities), the paragraph pasted below on calculating specific intestinal permeability from experimentally measured values, e.g. from Caco-2-cell permeability assays, should be removed from the documentation. This is not part of PK-Sim and may lead to misunderstandings (as in https://github.com/Open-Systems-Pharmacology/Forum/issues/668).

> In addition to the calculated specific intestinal permeability, experimentally determined permeabilities, e.g. from Caco-2-cell permeability assays can be used. However, due to the large inter-laboratory variability in Caco-2 permeations, a proper calibration of the measured in vitro values and the calculated in silico permeabilities for a defined set of compounds is necessary. If experimentally determined values for intestinal permeabilities are available and the customized calibration method has been implemented in PK-Sim®, this option is then available in the drop-down menu in the Calculation methods window.

**Comment by StephanSchaller** [comment]
I agree with Yuri, the paragraph is not wrong per see, it might just need a bit of more detail / be more specific and a minor correction:

The "calculation methods" window it refers to is actually the "Parameter Alternative" list in the Compound settings when you CREATE or CONFIGURE your simulations:
![image](https://user-images.githubusercontent.com/25477643/104822678-9eca2380-5844-11eb-8b50-d282fa85e956.png)

**Comment by AndreDlm** [comment]
Also fine for me, but then I would suggest to add a description for the user on how to conduct the correlation and come up with meaningful values for the calibration parameters...
